config/agent/agent.conf

```diff
@@ -12,5 +12,5 @@ NodeName=
 
 #
 # The IP address that hirte-agent can use to connect to hirte manager. It must be a valid IPv4 or IPv6 address.
-# It's mandatory to set this option for each hirte-agent.
+# It's mandatory to set either ManagerHost or ManagerAddress option for each hirte-agent.
```

Comment:
If `ManagerHost` is set, then also `ManagerPort` is required. Maybe this should be mentioned? And also mention the precedence of the address option. 
```
It's mandatory to set either ManagerHost and ManagerPort or ManagerAddress option for each hirte-agent. If both are set, ManagerAddress has precedence.  
```

Reply:
ManagerPort is set by default, so no need to set it (only if you want to override port)